README.md

```diff
@@ -1,3 +1,34 @@
-# Uploadcare Binding for PubNub Chat Engine
+# Uploadcare Plugin for Chat Engine
 
-Easily add image uploads to chatrooms using UploadCare online service.
+Adds the ability to upload images in ChatEngine.Chats via UploadCare service
+
+### Quick Start
+
+0. Have ChatEngine instantiated and connected, and have a channel you want to upload images on
+```javascript
+const CE = ChatEngineCore.create({
```

Comment:
Would personally prefer the constant to be called ```ChatEngine``` for branding purposes. It seems weird that it's called ChatEngine but there are no variables called that.

Maybe we should just move from ```ChatEngineCore``` to ```ChatEngine``` and then use ```CE``` as the preferred variable in apps.

Reply:
I was thinking the same thing, `ChatEngineCore` looks out of place.

Reply:
Thought about this more and still agree.

```
Library: ChatEngineCore -> ChatEngine
Examples: ChatEngine -> CE
```